app/settings/surveys/attribute-editor.html

```diff
@@ -32,7 +32,11 @@
    <!-- editing/adding categories -->
     <div class="form-field" ng-if="editAttribute.input ==='tags'">
       <label translate>Which labels should be available</label>
-        <div ng-repeat="category in availableCategories">
+        <div 
+          class="form-field checkbox"
+          ng-repeat="category in availableCategories"
+          ng-if="category.parent_id === null"
```

Comment:
Hrm. Might be cleaner to use `_.where(tags, { parent_id : null })` in the  directive instead